src/bionics.cpp

```diff
@@ -958,6 +958,27 @@ bool player::install_bionics( const itype &type, int skill_level )
         return false;
     }
 
+    const int pk = get_painkiller();
+    int pain_cap = 100;
+    if( has_trait( "PAINRESIST_TROGLO" ) ) {
+        pain_cap = roll_remainder( pain_cap * 0.5f );
```

Comment:
Drop the roll remainder thing: this is an UI restriction, meaning that attempting it takes no time. There should be no rolls in UI.